core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala

```diff
@@ -47,16 +47,22 @@ object DruidTuning
     *
     * @param maxRowsInMemory           number of rows to aggregate before persisting
     * @param intermediatePersistPeriod period that determines the rate at which intermediate persists occur
+<<<<<<< HEAD
```

Comment:
We have to have automatic travis tests for all our projects to catch situations like this automatically